simtools/util/general.py

```diff
@@ -48,12 +50,14 @@ def fileHasText(file, text):
     -------
     bool
     """
-    with open(file, "r") as ff:
-        for ll in ff:
-            if text in ll:
+    with open(file, "rb",0) as stringFile, \
+        mmap.mmap(stringFile.fileno(), 0, access=mmap.ACCESS_READ) as textFileInput:
+            re_search_1 = re.compile(br'.*(%s).*'%text.encode())
```

Comment:
Do you really need the `.*` around the text?

In addition, we should discuss style tomorrow, but in general use use either the `format` convention to build strings or `f-string` (the latter is preferred).

There's also a question of style of how to break the previous line, but we need to test it cause breaking `with` is always annoying.